app/assets/stylesheets/forever_style_guide/modules/_modal.scss

```diff
@@ -13,10 +14,26 @@ $border-radius-modal: 6px;
   border: none;
   border-top-right-radius: inherit;
   border-top-left-radius: inherit;
+  padding: 10px;
+
+  &.modal-promo-header {
```

Comment:
The SMACSS namespacing convention could be improved here by implmenting this as modal-header-promo (module-submodule-modifier)